src/app/modules/shared/containers/booking/booking.component.html

```diff
@@ -0,0 +1,28 @@
+<table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
+  <ng-container matColumnDef="startDate">
+    <th mat-header-cell *matHeaderCellDef>Start date</th>
```

Comment:
You should write "Start date" in all lowercase letters and add the style in CSS.